app/actions/remote/team.ts

```diff
@@ -428,3 +481,32 @@ export async function handleKickFromTeam(serverUrl: string, teamId: string) {
         logDebug('Failed to kick user from team', error);
     }
 }
+
+export async function getTeamMembersByIds(serverUrl: string, teamId: string, userIds: string[], fetchOnly?: boolean) {
+    let client;
+
+    try {
+        client = NetworkManager.getClient(serverUrl);
+        const {operator} = DatabaseManager.getServerDatabaseAndOperator(serverUrl);
+        const members = await client.getTeamMembersByIds(teamId, userIds);
+
+        if (!fetchOnly) {
+            const roles = [];
+
+            for (const {roles: memberRoles} of members) {
+                roles.push(...memberRoles.split(' '));
+            }
+
+            fetchRolesIfNeeded(serverUrl, Array.from(new Set(roles)));
+
+            if (operator) {
+                await operator.handleTeamMemberships({teamMemberships: members, prepareRecordsOnly: true});
+            }
```

Comment:
`operator` is always defined here.

```suggestion
            await operator.handleTeamMemberships({teamMemberships: members, prepareRecordsOnly: true});
```

Reply:
Done